src/group/components/GroupDefaultHomeCard.js

```diff
@@ -18,49 +18,40 @@ import React from 'react';
 
 import { useTranslation } from 'react-i18next';
 
-import { Alert, Box, Divider, Typography } from '@mui/material';
-
-import CardActionRouterLink from 'common/components/CardActionRouterLink';
+import { Box, Link, Typography } from '@mui/material';
 
 import HomeCard from 'home/components/HomeCard';
 
 const GroupDefaultHomeCard = () => {
   const { t } = useTranslation();
+  const groupsImage = require(`assets/${t(`group.img.src`)}`);
```

Comment:
Question: can you require this image outside the `GroupDefaultHomeCard` component? I think that way we will avoid executing this line multiple times

Reply:
I don't think `require()` can take runtime variables: https://github.com/facebook/react-native/issues/6391 for reference.

Reply:
Ohh, I didnt see that t usage, that makes sense